rsc/src/main/scala/rsc/outline/Synthesizer.scala

```diff
@@ -702,6 +683,16 @@ final class Synthesizer private (
     val method = DefnMethod(Mods(Nil), id, Nil, paramss, ret, rhs)
     scheduler(env, method.withPos(tree.pos))
   }
+
+  private def synthesizeTparam(tp: TypeParam): TypeParam = {
+    val mods = Mods(Nil)
+    val id = TptId(tp.id.valueopt.getOrElse("_")).withPos(tp.id.pos)
+    val lbound = tp.lbound.map(_.dupe)
+    val ubound = tp.ubound.map(_.dupe)
+    val innerTparams = tp.tparams.map(synthesizeTparam)
```

Comment:
Curious: which test case demonstrates the problem that requires recursion on type parameters?

Reply:
Talked offline. Higher-kinded types, as in `288.scala`.